check_release_assets.py

```diff
@@ -14,18 +14,26 @@
 CP39 = 'cp39-cp39'
 CP310 = 'cp310-cp310'
 LINUX = 'manylinux2014_x86_64'
+LINUX_V11 = 'manylinux1_x86_64'
 LINUX_AARCH64 = 'manylinux2014_aarch64'
 WINDOWS = 'win_amd64'
 
 
 sdist_project = 'cupy'
 
-_v11_cuda_matrix = list(itertools.product(
+_v12_cuda_matrix = list(itertools.product(
     (CP37, CP38, CP39, CP310), (LINUX, WINDOWS)))
+_v12_aarch64_matrix = list(itertools.product(
+    (CP37, CP38, CP39, CP310), (LINUX_AARCH64,)))
+_v12_rocm_matrix = list(itertools.product(
+    (CP37, CP38, CP39, CP310), (LINUX,)))
+
+_v11_cuda_matrix = list(itertools.product(
+    (CP37, CP38, CP39, CP310), (LINUX_V11, WINDOWS)))
 _v11_aarch64_matrix = list(itertools.product(
     (CP37, CP38, CP39, CP310), (LINUX_AARCH64,)))
 _v11_rocm_matrix = list(itertools.product(
-    (CP37, CP38, CP39, CP310), (LINUX,)))
+    (CP37, CP38, CP39, CP310), (LINUX_V11,)))
```

Comment:
This PR changes also v11 matrices. Is it intended?

Reply:
Yes. v12 (`LINUX`) will use `manylinux2014` (#282). v11 keep using `manylinux1`.